content/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics.md

```diff
@@ -47,3 +47,12 @@ You can search for repositories that are associated with a particular topic. For
   ![Form to enter topics](/assets/images/help/repository/add-topic-form.png)
 4. After you've finished adding topics, click **Save changes**.
   !["Save changes" button in "Edit repository details"](/assets/images/help/repository/edit-repository-details-save-changes-button.png)
+  
+## Limitations and quotas
+
+The following limitations apply to topics:
+
+- Topic names can only include basic alphanumeric characters.
+- Topic names can NOT include capital letters.
+- Topic names can be no longer than 50 characters.
```

Comment:
```suggestion
- use 50 characters or less.
```
Let's match this language to the banner that pops up when you try to use characters that aren't allowed.